tests/fuzz/s2n_certificate_extensions_parse_test.c

```diff
@@ -54,22 +94,60 @@ int LLVMFuzzerInitialize(const uint8_t *buf, size_t len)
 
 int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
 {
-    for (int version = 0; version < s2n_array_len(TLS_VERSIONS); version++) {
+    /* We need at least one byte of input to set parameters */
+    if (len != 0) {
 
+        /* Setup */
         struct s2n_stuffer fuzz_stuffer = {0};
-        GUARD(s2n_stuffer_alloc(&fuzz_stuffer, len + 1));
+        GUARD(s2n_stuffer_alloc(&fuzz_stuffer, len));
         GUARD(s2n_stuffer_write_bytes(&fuzz_stuffer, buf, len));
 
-        /* Setup */
         struct s2n_connection *client_conn = s2n_connection_new(S2N_CLIENT);
         notnull_check(client_conn);
-        client_conn->actual_protocol_version = TLS_VERSIONS[version];
-        client_conn->client_protocol_version = TLS_VERSIONS[version];
+
+        /* Pull a byte off the libfuzzer input and use it to set parameters */
+        uint8_t randval = 0;
+        GUARD(s2n_stuffer_read_uint8(&fuzz_stuffer, &randval));
+        client_conn->x509_validator.skip_cert_validation = (randval >> 7) % 2;
```

Comment:
We can fix this in a later PR, but I think we should create a Macro for this. `S2N_GET_BIT(randval, 7)` or similar.